PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -172,6 +172,56 @@ def miniAOD_customizeCommon(process):
     del process.slimmedMETsNoHF.caloMET
     # ================== NoHF pfMET
 
+    #  ==================  CHSMET 
+    process.CHSCands = cms.EDFilter("CandPtrSelector",
+                                    src=cms.InputTag("packedPFCandidates"),
+                                    cut=cms.string("fromPV(0) > 0")
```

Comment:
following up on today's discussion and last week mail thread, are we sure we want to use this definition for TkMET?
I'd say it makes little sense and we should rather go for a strict PV association (in the end people using TkMET want to know the MET @ vertex, that is PU stable, and they do not care of precise response as the missing neutral part is more relevant)

how about using this definition
https://github.com/cms-nanoAOD/cmssw/blob/fae71a4157a1b2aab3fb95d79bf7ac9f5eba918b/PhysicsTools/NanoAOD/python/met_cff.py#L8-L14

Reply:
given that we do not have a better definition right now (studies started), we keep it as is, and we can systematically update when we decide. This definition is hand in hand correlated with what jets and is used in the JEC computation. therefore the met has to be built from the pf candidates identical to what jets are reconstructed with.